src/System.Management.Automation/FormatAndOutput/DefaultFormatters/PowerShellCore_format_ps1xml.cs

```diff
@@ -739,120 +739,105 @@ private static IEnumerable<FormatViewDefinition> ViewsOf_System_Management_Autom
 
```

Comment:
@KirkMunro, your last commit had 1 failures in `PowerShell-CI-windows`
Write-Error Tests.ErrorRecord should not be truncated or have inserted newlines when redirected from another process
```powershell
Expected exactly 4, but got 3.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\Write-Error.Tests.ps1: line 106
106:         $result.Count | Should -BeExactly 4
```